app/views/videos/show.html.haml

```diff
@@ -4,48 +4,47 @@
--#          %a.btn.btn-default(href="") + My Queue
+        %br
+        %br
+        .actions
+          %a.btn.btn-primary(href="") Watch Now
+          %a.btn.btn-default(href="") + My Queue
 
--#%section.reviews.container
--#  .row
--#    .col-sm-10.col-sm-offset-1
--#      %form
--#        %fieldset
--#          .form-group
--#            %label Rate this video
--#            .row
--#              .col-sm-3
--#                %select.form-control(name="")
--#                  %option(value="5") 5 Stars
--#                  %option(value="4") 4 Stars
--#                  %option(value="3") 3 Stars
--#                  %option(value="2") 2 Stars
--#                  %option(value="1") 1 Star
--#          .form-group
--#            %label Write Review
--#            .row
--#              .col-sm-8
--#                %textarea.form-control(name="" rows="6")
--#        %fieldset.form-group.actions.clearfix
--#          %input(type="submit" value="Submit" class="btn")
--#          %a(href="") Cancel
--#      %header
--#        %h3 User Reviews (253)
--#      %ul
--#        - 8.times do
--#          %article.review
--#            %li.row
--#              .col-sm-2
--#                %span Rating: 5 / 5
--#                %p by <a href="">John A. Zoidberg</a>
--#              .col-sm-8
--#                %p In my opinion, this is one of the best shows ever made. It's not only funny, but despite being so frequently silly, it's very smart as well. Math, science, history, all get referenced. The best parts of the show are the subtle things, those little things in the background or just on screen for a couple of seconds that just make you laugh out loud if you were paying attention. The writers even appear to have been thinking ahead, because if you play close attention to the first episode, you can see a literal shadow of an event revealed seasons later.
--#
+%section.reviews.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      = form_for [@video, Review.new] do |f|
+        %fieldset
+          .form-group
+            = f.label "Rate this video"
+            .row
+              .col-sm-3
+                = f.select :rating,
+                  options_for_select([5,4,3,2,1].map { |number| [pluralize(number, "Star")]}),
+                  {},
+                  class: 'form-control'
+          .form-group
+            = f.label "Write Review"
+            .row
+              .col-sm-8
+                = f.text_area :description, rows: 6, class: "col-sm-10"
+        %fieldset.form-group.actions.clearfix
+          = f.submit "Submit", class: "btn"
+          = link_to "Cancel", video_path(@video)
+      %header
+        %h3= "User Reviews (#{@video.reviews.size})"
+      %ul
+        - @reviews.each do |review|
+          %article.review
+            %li.row
+              .col-sm-2
+                %span= "Rating: #{review.rating}"
+                %p= "by #{link_to review.user.full_name, ''}".html_safe
```

Comment:
Be careful using html safe. Theoretically this is unsafe since it contains user input. You can do this without the interpolation. Just put the link on a separate line from "by".